src/jesse_state.erl

```diff
@@ -36,9 +36,10 @@
         , set_allowed_errors/2
         , set_current_schema/2
         , set_error_list/2
-        , find_schema/2
```

Comment:
The jesse API is the one defined in `jesse.erl`.

If you have a usecase, let me know and we can look into this as well, otherwise I can't work with "what ifs", and pollute the codebase with aliases and/or backward-compatibility wrappers because maybe somewhere someone somehow would depend on x and they cannot be bothered to run `sed "s/x/y/g"`.